### PR TITLE
Importer: Clean up section-import

### PR DESCRIPTION
**Pull request by dmsnell**
- Trim constants via ES6 module exports/imports
 - Merge render logic into `render()`
 - General cleanup

**Testing** This shouldn't change any functionality, so if you attempt an import and something borks out, it's broken. Otherwise it should be coding changes only.

<del>The PR depends on #2949 and will shrink considerably once that's merged in.</del>

cc: @apeatling @rodrigoi @omarjackman @dllh @designsimply

### CODE REVIEW

**Comment by dmsnell** [comment]
cc: @rralian

**Comment by dllh** [comment]
I gave this a quick test drive and discovered no regressions. Would prefer to leave code review to others. Maybe @omarjackman or @jordwest could take a peek?

File: server/devdocs/Makefile [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-BASE_DIR := $(NODE_BIN)/../..
-NODE_PATH := test:$(BASE_DIR)/client
-COMPILERS ?= js:babel/register
-REPORTER ?= spec
-UI ?= bdd
-
-test:
-	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers $(COMPILERS) --reporter $(REPORTER) --ui $(UI)
-
-.PHONY: test
```
Comment (jordwest) [inline server/devdocs/Makefile]: Was this file intentionally deleted?

Comment (dmsnell) [inline server/devdocs/Makefile]: it was left over from another PR where it _should_ have been deleted. there are no tests left so the Makefile should go too

**Comment by jordwest** [comment]
Gave it a test locally, also couldn't see any regressions. Code changes LGTM :ship:

**Pull request merged**